index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Tribute Page</title>
+  </head>
+  <body>
+    <main id="main">
+      <h1 id="title">Dr. Norman Borlaug</h1>
+      <div id="img-div">
+        <img
+          id="image"
+          src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg"
+          alt="image" />
+        <div id="img-caption">
+          Dr. Norman Borlaug, third from the left, trains biologists in Mexico
+          on how to increase wheat yields - part of his life-long war on hunger.
+        </div>
+        <section id="tribute-info">
+          <h2>Here's a time line of Dr. Borlaug's life:</h2>
+          <p>1914 - Born in Cresco, Iowa</p>
+          <p>
+            1933 - Leaves his family's farm to attend the University of
+            Minnesota, thanks to a Depression era program known as the "National
+            Youth Administration"
+          </p>
+          <p>
+            1935 - Has to stop school and save up more money. Works in the
+            Civilian Conservation Corps, helping starving Americans. "I saw how
+            food changed them", he said. "All of this left scars on me."
+          </p>
+          <p>
+            1937 - Finishes university and takes a job in the US Forestry
+            Service
+          </p>
+          <p>
+            1938 - Marries wife of 69 years Margret Gibson. Gets laid off due to
+            budget cuts. Inspired by Elvin Charles Stakman, he returns to school
+            study under Stakman, who teaches him about breeding pest-resistent
+            plants.
+          </p>
+          <p>
+            1941 - Tries to enroll in the military after the Pearl Harbor
+            attack, but is rejected. Instead, the military asked his lab to work
+            on waterproof glue, DDT to control malaria, disinfectants, and other
+            applied science.
+          </p>
+          <p>1942 - Receives a Ph.D. in Genetics and Plant Pathology</p>
+        </section>
+        <a
+          id="tribute-link"
+          href="https://en.wikipedia.org/wiki/Norman_Borlaug"
+          target="_blank"></a>
```

Comment:
Wrap your `<a>` in a `<footer></footer>` if you wish your link to appear at the end of the page otherwise please wrap it inside your `section` tag

Reply:
- [ ] Hi @ashmalzahra , Good job :+1:  for using [GitHub inline comments feature](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request) for suggested changes, But please kindly makes sure that you re-phrase the comments politely, as suggestions instead of orders and includes an explanation, a "why", for each requested change.